lib/Test/WWW/Stub/Intercepter.pm

```diff
@@ -12,7 +12,7 @@ sub new {
 
 sub intercept {
     my ($self, $uri, $env, $req) = @_;
-    for my $pattern (keys %{ $self->{registry} }) {
+    for my $pattern (sort { length $a <=> length $b } keys %{ $self->{registry} }) {
```

Comment:
Can you add some tests?

Reply:
🙇 I added tests: https://github.com/astj/p5-Test-WWW-Stub/pull/20/commits/fff84e84e76466d43efb96eab95b3af4f2d6fd1e